client/luigi-client.js

```diff
@@ -188,7 +188,7 @@
       return false;
     },
     /**
-     * Registers a listener that is called upon any navigation change.
+     * Registers a listener that is called with a context object upon any navigation change.
```

Comment:
```suggestion
     * Registers a listener that is called with the context object upon any navigation change.
```
I have noticed we are  linking to a general {Object} definition throughout the document, but not in the case of context object - is it because it's a particular instance of the {Object}? I'm absolutely not saying it's wrong - just want to know :)

Below we have also mentioned the context object (Returns the context object) and a line below we have {Object} current context data.

Reply:
The context object could contain any parameter, it reflects the state of the microfrontend